### DISCUSSION TITLE
Managed Redis

### DISCUSSION THREAD

**Discussion by bgervan** [ubicloud/ubicloud#1707] [Ideas]
Is there a plan to support redis (or alternative like dragonflydb) as managed service with replica etc?

**Comment by byucesoy** [comment]
Hi @bgervan,

We are planning to add new services and redis (or one of its alternatives) could be one of them. However at this time, there are few higher priority services that we want to release first, so I think redis would not arrive anytime soon.

Depending on our users ask, priority of managed redis can increase, so anyone who is reading this and interested in managed redis, feel free to leave thumbs up. Also, we would appreciate if you write few sentences about how you use/plan to use redis and what is your expectations from a managed redis service. It would help us a lot while designing our managed redis service.

**Reply by bgervan** [reply]
Thank you for the quick reply.

Many system relies on redis as cache and also for job backends (ie Celery with python). For the second, it's important to have some kind of failover or the system can try to send job requests and lost data.

So my primary use case would be to have a replica which can be put in place if the master dies for some reason. For k8s, there are options, but if we want to change to ubicloud and kamal, the only alternative to use a different job backend like nats and build a nats jetstream cluster, in order to achieve HA without the kubernetes management overhead.

**Reply by bgervan** [reply]
Btw, is there a public roadmap to see what's in the tunnel?

**Reply by ozgune** [reply]
Hi @bgervan, thank you for sharing the above feedback!

We do our product planning every six weeks and we don't maintain a public roadmap (yet). We'll have our next planning session in a week; and elastic load balancers and managed K8s are currently the top contenders.
